pkg/qtv/protocol_writer.go

```diff
@@ -139,9 +142,14 @@ func (qp *qProtocol) putBaseLines(msg *netMsgW, cursize int, maxbuffersize int,
 		}
 
 		if qp.baseLine[i].modelIndex != 0 {
-			msg.PutSVC(svc_spawnbaseline)
-			msg.PutUint16(uint16(i))
-			qp.putEntityState(msg, &qp.baseLine[i])
+			if (qp.extFlagsFTE1 & ftePextSpawnStatic2) != 0 {
+				msg.PutSVC(svc_fte_spawnbaseline2)
+				qp.putDelta(msg, i, &nullEntityState, &qp.baseLine[i], true)
+			} else {
```

Comment:
matter of taste here, could also remove the non-spawnstatic2 versions completely.